zebra-state/src/service/finalized_state/disk_format/transparent.rs

```diff
@@ -416,36 +416,36 @@ impl AddressTransaction {
         }
     }
 
-    /// Create an [`AddressTransaction`] which starts iteration for the supplied
+    /// Create a range of [`AddressTransaction`]s which starts iteration for the supplied
     /// address. Starts at the first UTXO, or at the `query_start` height,
-    /// whichever is greater.
+    /// whichever is greater. Ends at the last transaction index for an [`AddressLocation`].
```

Comment:
This documentation is not quite right: the range ends at the end height or the maximum possible transaction index.

This is different from the last actual UTXO, which is stored in the database and requires an iteration query to discover.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/7732/commits/cb9a1f66c53758597329b2f36278df3c8126b818.